pkg/sql/alter_database.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package sql
+
+import (
+	"context"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/catalogkv"
+	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgcode"
+	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/sql/sqlbase"
+)
+
+type alterDatabaseOwnerNode struct {
+	n    *tree.AlterDatabaseOwner
+	desc *sqlbase.MutableDatabaseDescriptor
+}
+
+// AlterDatabaseOwner transforms a tree.AlterDatabaseOwner into a plan node.
+func (p *planner) AlterDatabaseOwner(
+	ctx context.Context, n *tree.AlterDatabaseOwner,
+) (planNode, error) {
+	immutableDbDesc, err := p.ResolveUncachedDatabaseByName(ctx, n.Name.String(), true /*required*/)
```

Comment:
add a TODO for lucy to switch those over to the mutable db access method once it lands

Reply:
Done